README.md

```diff
@@ -29,6 +29,8 @@ Install CLI tool → http://cli.scalingo.com
 If you have `package.json` file in your repository, Node.JS will be installed automatically
 and dependencies will be installed according to the `package.json` file.
 
+You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`
```

Comment:
```suggestion
You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`.
```

Is it possible to explain the cases where one would need to disable this feature?